### ISSUE TITLE
Bad readings and retry

### ISSUE THREAD

**Issue by kaputtzich** [jervine/rpi-temp-humid-monitor#2]
Is it possible to ignore a specific temp reading if it's wrong?
I use a DHT22 in my office and sometimes there's a bad reading of -0.1.
The script just stops further database writing because of that.

Here' a short snippet of the log file

Damn ... the sensor returned a bad reading!
-Retry--Retry--Retry--Retry--Retry-Current Temp:  -0.1, Previous Temp:  25.3, Temperature Difference: -25.4C
Current Temp:  25.2, Previous Temp:  -0.1, Temperature Difference:  25.3C
Damn ... the sensor returned a bad reading!
Current Temp:  25.2, Previous Temp:  -0.1, Temperature Difference:  25.3C
Damn ... the sensor returned a bad reading!
-Retry-Current Temp:  25.2, Previous Temp:  -0.1, Temperature Difference:  25.3C
Damn ... the sensor returned a bad reading!

**Comment by kaputtzich** [comment]
Set the delay time to 60 seconds, maybe this helps if the sensor is to slow

**Comment by jervine** [comment]
Sorry for the delay in responding - there must be an error in my code. I'll try and find time in the next few days to amend this.

**Comment by jervine** [comment]
Sorry, been a _long_ time since I've looked at this ... is this still a problem? The underlying problem, as I see it, is that the broken reading of -0.1 has actually been committed to the database. This means that subsequent (correct) readings are too far out and are therefore rejected. I had built in tolerances of changes that were allowed, so I'm a bit puzzled why -0.1 was allowed (maybe my coding was not up to scratch and allowed a negative value through).

If this is still a problem, I can try and have a look again.
